src/components/Settings/ShareOnSocialMedia.js

```diff
@@ -55,56 +56,91 @@ const TelegramShareButton = styled(_TelegramShareButton)`
   ${commonIconStyle};
 `;
 
-const ShareOnSocialMedia = () => {
-  const shareUrl = 'https://susi.ai';
-  const title = 'Lets chat with SUSI.AI, the open source personal assistant';
-  const shareText =
-    'Lets chat with SUSI.AI, the open source personal assistant https://susi.ai';
+const shareUrl = 'https://susi.ai';
+const shareText =
+  'Lets chat with SUSI.AI, the open source personal assistant https://susi.ai';
 
-  return (
-    <React.Fragment>
-      <DialogTitle>Share about SUSI</DialogTitle>
+class ShareOnSocialMedia extends Component {
+  state = {
+    shareUrl,
+    title:
+      this.props.title ||
+      'Lets chat with SUSI.AI, the open source personal assistant',
```

Comment:
shareText and the title string in common. Please reuse them by shareText + url.

Reply:
Done sir!